src/main/msp/msp.c

```diff
@@ -3238,6 +3251,28 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         break;
 #endif
 
+#ifdef USE_SBUS_OUTPUT
+    case MSP_SET_SBUS_OUTPUT_CONFIG: {
+        // Write format is customized for the size and responsiveness.
+        // The first byte will be the target output channel index (0-based).
+        // The following bytes will be the type/index/low/high for that channel.
+        // `frameRate` is piggyback to any (all) indices for simplicity.
+        if (sbufBytesRemaining(src) >= 1) {
+            uint8_t index = sbufReadU8(src);
+            if (index < SBUS_OUT_CHANNELS && sbufBytesRemaining(src) >= 7) {
+                sbusOutConfigMutable()->sourceType[index] = sbufReadU8(src);
+                sbusOutConfigMutable()->sourceIndex[index] = sbufReadU8(src);
+                sbusOutConfigMutable()->sourceRangeLow[index] = sbufReadS16(src);
+                sbusOutConfigMutable()->sourceRangeHigh[index] = sbufReadS16(src);
+
+                // You need to reset FC after updating ->frameRate.
+                sbusOutConfigMutable()->frameRate = sbufReadU8(src);
```

Comment:
I am not convinced that repeating the framerate parameter with every channel is the right thing to do.

Reply:
I have coded the unto deal with it... But maybe a better option is to not do msp frame rate changes.. And make that cli only?

Reality is that the default frame rate is probably perfect in 99.99% of cases.

Reply:
Let's remove the frameRate from this MSP call. It's not correct to do it this way.

Reply:
> I am not convinced that repeating the framerate parameter with every channel is the right thing to do.
It's just a compromise of simpleness and cleanness. I feel it's not super necessary to create a separate protocol for a single uncommon byte.

> I have coded the unto deal with it... But maybe a better option is to not do msp frame rate changes.. And make that cli only?
> 
> Reality is that the default frame rate is probably perfect in 99.99% of cases.
I think we should keep all tunables available through MSP, so that when ready we can configure everything from the radio.

> Remove frameRate from this MSP. Then it does not need reboot either.
This isn't solving the reboot issue. Reboot is not needed if I don't change the frameRate. It is needed when changed, regardless where (other MSP or cli). 
This is because frameRate is read on init. The way to improve is to add `rescheduleTask` here too but this doesn't seem to follow other MSP conventions (MSP shouldn't care how the params are used)